src/SFA.DAS.EmployerIncentives.Functions.LegalEntities/Services/LegalEntities/IVendorRegistrationFormService.cs

```diff
@@ -5,6 +5,6 @@ namespace SFA.DAS.EmployerIncentives.Functions.LegalEntities.Services.LegalEntit
 {
     public interface IVendorRegistrationFormService
     {
-        Task Update(DateTime fromDateTime);
+        Task<DateTime> Update(DateTime fromDateTime);
```

Comment:
It's not clear from this interface what is being returned.  I call Update passing a date and I get another date back.  Normally I wouldn't expect to get a response back from an update (other than an exception if it failed).  Is the function doing work that should/could be in the inner application layer?

Reply:
it's storing a date returned by APIM endpoint.
we have a Jira ticket to move the logic into Inner Api